lib/Doctrine/ORM/Sequencing/SequenceGenerator.php

```diff
@@ -70,41 +64,31 @@ public function generate(EntityManagerInterface $em, $entity)
 
     /**
      * Gets the maximum value of the currently allocated bag of values.
-     *
-     * @return int|null
      */
-    public function getCurrentMaxValue()
+    public function getCurrentMaxValue() : ?int
     {
         return $this->maxValue;
     }
 
     /**
      * Gets the next value that will be returned by generate().
-     *
-     * @return int
      */
-    public function getNextValue()
+    public function getNextValue() : int
     {
         return $this->nextValue;
     }
 
-    /**
-     * @return string
-     */
-    public function serialize()
+    public function serialize() : string
     {
         return serialize(
             [
-            'allocationSize' => $this->allocationSize,
-            'sequenceName'   => $this->sequenceName,
+                'allocationSize' => $this->allocationSize,
+                'sequenceName'   => $this->sequenceName,
             ]
         );
     }
 
-    /**
-     * @param string $serialized
-     */
-    public function unserialize($serialized)
+    public function unserialize($serialized) : void
```

Comment:
Can a `string` type be added here? Or does it collide with `Serializable`?

Reply:
It collides with `Serializable`

Reply:
I'll put the docblock back, my intention was to add the type declaration

Reply:
done